### PR TITLE
increment version to 0.18.0

### PR DESCRIPTION
**Pull request by albi3ro**

### CODE REVIEW

**Comment by Jaybsoni** [comment]
Do we need to update the setup.py file to include the new version # ?

**Review by Jaybsoni — CHANGES_REQUESTED** [review]
Looks good to me, not sure if we need to update the setup.py file, aside from that I think its good to go!

File: pennylane_qiskit/qiskit_device.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ class QiskitDevice(QubitDevice, abc.ABC):
             to simulate a device compliant circuit, you can specify a backend here.
     """
     name = "Qiskit PennyLane plugin"
-    pennylane_requires = ">=0.17.0"
+    pennylane_requires = ">=0.18.0"
```
Comment (antalszava) [inline pennylane_qiskit/qiskit_device.py:88]:
```suggestion
    pennylane_requires = ">=0.17.0"
```

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 qiskit>=0.25
-pennylane>=0.17
+pennylane>=0.18
```
Comment (antalszava) [inline requirements.txt:2]:
```suggestion
pennylane>=0.17
```

**Review by antalszava — CHANGES_REQUESTED** [review]
Hi @albi3ro, @Jaybsoni, we only every increment the PennyLane version number in the `requirements.txt` and `setup.py` files and the `pennylane_requires` attribute in the device if there's a main feature in the new version of PennyLane (`v0.18.0`) that the plugin depends on.

This is not the case here, as the only change since the last release in the plugin was removing a deprecated keyword argument.

**Review by Jaybsoni — APPROVED** [review]
Ok, so all that was changed was the changelog and the _version.py file. 
Thanks for the review @antalszava!

**Review by antalszava — APPROVED** [review]

**Pull request merged**